dashboard/visualization/text_tables.py

```diff
@@ -218,3 +217,29 @@ def pca_summary(pca: PCA, activation_columns: list[str]):
     )
 
     return projection_info
+
+
+def pca_smiles_summary() -> html.Details:
+    projection_smiles_info_text = """To calculate the projections, Extended-Connectivity Fingerprints (ECFP) were used.
+    Fingerprints were calculated using RDKit method """
+
+    text_with_hyperlink = html.Span(
+        [
+            projection_smiles_info_text,
+            html.A(
+                "GetMorganFingerprintAsBitVect",
+                href="https://www.herongyang.com/Cheminformatics/Fingerprint-RDKit-Morgan-GetMorganFingerprintAsBitVect.html",
+                target="_blank",
+            ),
+        ]
+    )
+
+    projection_smiles_info = html.Details(
+        [
+            html.Summary(html.Strong("ADDITIONAL PROJECTION INFORMATION")),
+            html.Ul(
+                text_with_hyperlink,
+            ),
+        ]
+    )
+    return projection_smiles_info
```

Comment:
Why is it defined here as a function and then injected by a callback instead of being statically defined in the `s2_smiles_display.py` stage in `div#pca-smiles-info`?

Reply:
Your'e right, that's completely unnecessary 🥴 moved it to `s2_smiles_display.py` as a static elem